lib/statistics.js

```diff
@@ -0,0 +1,51 @@
+import mongo from './util/mongo.js'
+
+export async function getStatistics() {
+  const statistics = await mongo.db.collection('statistics').find().sort({date: 1}).toArray()
+  const last30Days = statistics.slice(-30)
+  const statsToReturn = {
+    subscriptions: 0,
+    profileRepartition: {},
+    departementRepartition: {},
+    regionRepartition: {},
+    statsByDay: []
+  }
+  for (const stat of last30Days) {
+    for (const profile in stat.profileRepartition) {
+      if (Object.hasOwn(stat.profileRepartition, profile)) {
+        statsToReturn.profileRepartition[profile] = statsToReturn.profileRepartition[profile]
+          ? statsToReturn.profileRepartition[profile] + stat.profileRepartition[profile]
+          : stat.profileRepartition[profile]
+      }
+    }
+
+    for (const departement in stat.departementRepartition) {
```

Comment:
Tu devrais utiliser `for..of` (sans le `Object.hasOwn` du coup)

Reply:
Done !